invenio_app_ils/ill/schemas/ill-borrowing-requests/borrowing-request-v1.0.0.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Schema for the ILL borrowing request",
+  "required": [
+    "$schema",
+    "pid",
+    "status",
+    "library_pid"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Interlibrary borrowing request schema",
+      "type": "string"
+    },
+    "pid": {
+      "title": "The persistent identifier of ILL borrowing request",
+      "type": "string"
+    },
+    "status": {
+      "title": "Status of the ILL borrowing request",
+      "type": "string"
+    },
+    "cancel_reason": {
+      "title": "Reason for cancelling the ILL borrowing request",
+      "type": "string"
+    },
+    "library_pid": {
```

Comment:
Should we have some info about the library as well? I am thinking in the details page of the Ill request maybe the library name is needed or a link would be enough?

Reply:
There is already the resolver for that. Is this what you mean?

Reply:
Added to the schema!